comms/src/connection/peer_connection/worker.rs

```diff
@@ -227,7 +227,7 @@ impl PeerConnectionWorker {
             }
 
             if let Ok(event) = monitor.read(1) {
-                self.handle_socket_event(event)?;
+                if self.handle_socket_event(event)? {}
```

Comment:
Is something going to end up here later? Otherwise this if seem like it could be removed?

Reply:
Oh good spot - thought I removed that